infra/cifuzz/cifuzz.py

```diff
@@ -460,3 +481,27 @@ def parse_fuzzer_output(fuzzer_output, out_dir):
   summary_file_path = os.path.join(out_dir, 'bug_summary.txt')
   with open(summary_file_path, 'a') as summary_handle:
     summary_handle.write(summary_str)
+
+
+def is_project_sanitizer(sanitizer, oss_fuzz_project_name):
+  """Finds all of the sanitizers a project can use for building and running.
+
+  Args:
+   sanitizer: The desired sanitizer.
+   oss_fuzz_project_name: The name of the relevant OSS-Fuzz project.
+
+  Returns:
+  True if project can use sanitizer.
+  """
+  project_yaml = os.path.join(helper.OSSFUZZ_DIR, 'projects',
+                              oss_fuzz_project_name, 'project.yaml')
+  if not os.path.isfile(project_yaml):
+    logging.error('project.yaml for project %s could not be found.',
+                  oss_fuzz_project_name)
+    return False
+
+  # Simple parse to prevent adding pyYAML dependency.
+  with open(project_yaml, 'r') as file_handle:
+    if sanitizer + '\n' in file_handle.read():
```

Comment:
This is not ideal. If anyone can think of a better lightweight way to do this without pyYAML let me know.

Reply:
Do we need this at all? Why do we need this to be a subset of the ones specified in project.yaml? We can just compare against a hardcoded list.

Reply:
> This is not ideal. If anyone can think of a better lightweight way to do this without pyYAML let me know.

Why exactly do we want to avoid using pyYAML?

Reply:
@jonathanmetzman I don't want to add any dependencies, but if that is ok with you it would make sense to add it. @oliverchang Isn't the project.yaml file essentially a hard coded list of every sanitizer a project supports? Where else could we get this info?

Reply:
What's the downside of dependencies? wasting time?

Reply:
Yes but I can't image that it is appreciable.

Reply:
I would just use pyyaml instead then.